nekoyume/Assets/_Scripts/BlockChain/ActionUnrenderHandler.cs

```diff
@@ -385,10 +385,12 @@ private void ResponseTransferAsset(ActionBase.ActionEvaluation<TransferAsset> ev
                 return;
             }
 
+            var senderAddress = eval.Action.Sender;
             var recipientAddress = eval.Action.Recipient;
             var currentAgentAddress = States.Instance.AgentState.address;
 
-            if (recipientAddress == currentAgentAddress)
+            if (recipientAddress == currentAgentAddress ||
```

Comment:
I think `recipientAddress == currentAgentAddress &&`
|| -> &&

Reply:
This should be rendered both sender and recipient, so I used `or` condition.

Reply:
ahh! i understand